com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/test_props.lua

```diff
@@ -20,8 +20,8 @@ go.property(invalid_string, 2)
 go.property("three_args", 1, 2)
 go.property("unknown_type", "hello")
 
-'go.property("no_prop", 123)'
-"go.property('no_prop', 123)"
+local s1 = 'go.property("no_prop", 123)'
+local s2 = "go.property('no_prop', 123)"
```

Comment:
There was a problem with the previous test code. `go.property("unknown_type", "hello") 'go.property("no_prop", 123)'` is parsed as a nested function call, as if `go.property()` returns a function and it then gets called with a string arg, like this:
```lua
go.property("unknown_type", "hello")('go.property("no_prop", 123)')("go.property('no_prop', 123)")
```
This makes `go.property` not a top-level form, and it's reported as error, while the test is testing that `go.proeprty` calls within strings are not parsed as `go.property` invocations